target/linux/x86/image/64.mk

```diff
@@ -7,4 +7,15 @@ define Device/generic
 	kmod-tg3
   GRUB2_VARIANT := generic
 endef
+
 TARGET_DEVICES += generic
+
+define Device/AZB_MT110
+  DEVICE_VENDOR := Intel Arizona Beach (NEXSEC MT110)
+  DEVICE_PACKAGES += \
+	kmod-e1000e kmod-e1000 kmod-fs-vfat kmod-igb kmod-ixgbe kmod-igc kmod-i40e \
+	kmod-iwlwifi kmod-usb-net-rndis kmod-usb-net-cdc-mbim kmod-i2c-i801
```

Comment:
Does this device has a kmod-i40e (Intel 40G) network card?

Reply:
MT110 is a Low Power,High Performance  network appliance. igc/igb in board , and i40e (10G/40G) for PCIe. We trying to run vpp/dpdk  on openwrt.

Reply:
> kmod-i2c-i801 kmod-i40e kmod-iwlwifi kmod-usb-net-cdc-mbim kmod-usb-net-rndis

I'm fine with adding those packages to generic target.

Reply:
Thanks! I will continue to improve.